internal/kldeth/txn.go

```diff
@@ -372,9 +384,20 @@ func genMethodABI(jsonABI *kldmessages.ABIMethod) (method *abi.Method, err error
 			err = klderrors.Errorf(klderrors.TransactionSendOutputTypeUnknown, i, jsonOutput.Name, err)
 			return
 		}
-		method.Outputs = append(method.Outputs, arg)
+		outputs[i] = arg
 	}
-	return
+
+	var abiMethod abi.Method
+	// Note we only support "normal" functions here - constructors do not follow this code path, and we don't have support
+	// of Fallback or Receive yet
+	switch jsonABI.Type {
+	case "function":
+	case "":
```

Comment:
Is this for `Fallback` function? If so, the comment preceding this is contradictory

Reply:
Actually, no. This is just to cover the (likely) case that when specifying the ABI in-line in YAML, the user does not include a `type` in there.